webapps/frontend/app/views/site/backend-alba-detail.html

```diff
@@ -470,6 +471,12 @@ <h4 class="pull-left" data-i18n="alba:generic.presets"></h4>
                                     <!-- /ko -->
                                 </td>
                                 <td>
+                                    <i data-bind="click: function() { if (!inUse) { $root.editPreset($data); } },
+                                                  style: { color: (!inUse) ? 'black' : 'lightgrey' },
+                                                  tooltip: (inUse || isDefault) ? 'alba:presets.cantdelete' : ''"
```

Comment:
That's probably the wrong tooltip. And is it correct you can't edit a preset that is in use?